README.md

```diff
@@ -1 +1,65 @@
-# React-Capstone-project-Financial-modeling
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Crypto currencies
```

Comment:
- [ ] Kindly record a video presentation of your project and include the link in the README.

- [ ] Kindly deploy your app and include the live demo link in your README.

- [ ] Kindly include unit tests for your individual components and integration tests using the React Testing Library.